src/sql/workbench/contrib/notebook/browser/cellViews/code.css

```diff
@@ -58,7 +65,10 @@ code-component .toolbarIconStop {
 }
 
 code-component .editor {
-	padding: 5px 0px 5px 0px
+	padding: 14px 0px 5px 0px
+}
+code-component .editor .line-numbers {
+	color: #A19F9D;
```

Comment:
We should try to use common colors from the color registry if possible instead of hardcoding so that they're theme-aware. If there aren't appropriate colors in there then you should add them so that themes can customize the colors. 

(let me know if you have questions on this)

Reply:
Thanks for the heads-up on the color registry.

Reply:
@Charles-Gagnon I added colors to the sql path colorRegistry and wired up registerThemingParticpant to the code component.